### DISCUSSION TITLE
The exported requirements file is no longer cross-platform. Using it on other platforms may cause unexpected result.

### DISCUSSION THREAD

**Discussion by anentropic** [pdm-project/pdm#2308] [Q&A]
When I use `pdm export` I get a message:

> The exported requirements file is no longer cross-platform. Using it on other platforms may cause unexpected result.

What does it actually mean?

Is there a way to export a requirements file that _is_ cross-platform?

**Comment by frostming** [comment]
> The exported requirements file is no longer cross-platform. Using it on other platforms may cause unexpected result.

This means the exported requirement file can only be used on the same platform and python version. You would expect incorrect dependencies on different ones.



> Is there a way to export a requirements file that _is_ cross-platform?

No for now.
